pynicotine/transfers.py

```diff
@@ -1180,9 +1178,9 @@ def file_upload_init(self, msg):
             if upload in self.transfer_request_times:
                 del self.transfer_request_times[upload]
 
-            real_path = self.core.shares.virtual2real(filename)
+            real_path = self.core.shares.virtual2real(filename, username)
```

Comment:
Why is it necessary for the `username` argument to be parsed from here, but is not required from userbrowse?